source/_data/plugins.yml

```diff
@@ -24,14 +24,19 @@
       description: Watches and bundles your spec files via webpack.
       link: https://github.com/cypress-io/cypress-webpack-preprocessor
       keywords: [webpack]
-      
+
+    - name: cypress-app-watcher-preprocessor
+      description: Reruns Cypress tests when the backend server restarts
```

Comment:
I'd probably say backend or frontend :) in most cases this will be actually more useful for reloading for example a create-react-app frontend. But it's cool that it reacts to either one!